### PR TITLE
Added sending updates for metadata update processing [#1076]

### LINKED ISSUE #1076: The scraping batch process publishes progress updates.
As a system, as the scraping process runs, messages are published with the status. These updates can be displayed on the browser side.

## Prerequisites
1. #1004

### PR DESCRIPTION
**Pull request by mcpierce**
Closes #1076 

## Status
READY

## Does this PR contain migrations?
NO

## Before You Submit Your PR:
- [ ] Have you announced your PR on the comixed-dev mailing list?
- [X] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactors existing code (code changes for efficiency or maintainability)
- [ ] Security fix (be sure to include the CVE in the commit message) 
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: comixed-batch/src/test/java/org/comixedproject/batch/metadata/listeners/ScrapeComicBookChunkListenerTest.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+    try {
+      listener.afterChunk(chunkContext);
+    } finally {
+      final MetadataUpdateProcessUpdate payload = payloadArgumentCaptor.getValue();
+      assertNotNull(payload);
+      assertTrue(payload.isActive());
+      assertEquals(TEST_TOTAL_COMICS.longValue(), payload.getTotalComics());
+      assertEquals(
+          TEST_TOTAL_COMICS - TEST_REMAINING_COMICS.longValue(), payload.getCompletedComics());
```
Comment (BRUCELLA2) [inline comixed-batch/src/test/java/org/comixedproject/batch/metadata/listeners/ScrapeComicBookChunkListenerTest.java:110]: unnecessary unboxing ?

File: comixed-batch/src/test/java/org/comixedproject/batch/metadata/listeners/ScrapeComicBookChunkListenerTest.java [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        .thenReturn(TEST_REMAINING_COMICS);
+
+    listener.afterChunk(chunkContext);
+
+    final MetadataUpdateProcessUpdate payload = payloadArgumentCaptor.getValue();
+    assertNotNull(payload);
+    assertTrue(payload.isActive());
+    assertEquals(TEST_TOTAL_COMICS.longValue(), payload.getTotalComics());
+    assertEquals(
+        TEST_TOTAL_COMICS - TEST_REMAINING_COMICS.longValue(), payload.getCompletedComics());
```
Comment (BRUCELLA2) [inline comixed-batch/src/test/java/org/comixedproject/batch/metadata/listeners/ScrapeComicBookChunkListenerTest.java:130]: unnecessary unboxing ?

File: comixed-batch/src/test/java/org/comixedproject/batch/metadata/listeners/ScrapeComicBookChunkListenerTest.java (hunk 1 above)
Comment (mcpierce) [reply comixed-batch/src/test/java/org/comixedproject/batch/metadata/listeners/ScrapeComicBookChunkListenerTest.java:110]: No. assertEquals() doesn't know what to do when it's a primitive and an object, like long and Long, being compared. So you have to be explicit and tell it what facet of the object to use for the comparison.

File: comixed-batch/src/test/java/org/comixedproject/batch/metadata/listeners/ScrapeComicBookChunkListenerTest.java (hunk 2 above)
Comment (mcpierce) [reply comixed-batch/src/test/java/org/comixedproject/batch/metadata/listeners/ScrapeComicBookChunkListenerTest.java:130]: See previous response.

**Pull request merged**